source/index.html

```diff
@@ -129,12 +129,16 @@ <h1 class="row-title" id="dinner">Top Dinner Recipes</h1>
         </div>
       </section>
 
-      <section class="search-results hidden">
+      <section class="search-results hidden" id="search-results-page">
         <!-- TODO: Populate query field -->
-        <h1>Top recipes for "bread"</h1>
-        <div class="results-grid">
+        <h1 id="search-results-title">Top recipes for "bread"</h1>
+        <div class="results-grid" id="results-grid-recipe-card-location">
           <!-- TODO: Populate recipe cards -->
         </div>
+        <div class="button--wrapper">
+          <img src="source/images/arrow-down.png" alt="Arrow down" />
+          <button id="show-more-button">Show more</button>
```

Comment:
for now, just keep the button (no div or img). we can let antonia and jon style it after